frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/labeling/SyncToRemoteStatusLabelGenerator.java

```diff
@@ -0,0 +1,25 @@
+package com.virtuslab.gitmachete.frontend.graph.labeling;
+
+import static com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus.Status.Ahead;
+import static com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus.Status.Behind;
+import static com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus.Status.Diverged;
+import static com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus.Status.Untracked;
+
+import java.util.Map;
+
+import com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus;
+
+public final class SyncToRemoteStatusLabelGenerator {
+  private SyncToRemoteStatusLabelGenerator() {}
+
+  private static final Map<SyncToRemoteStatus.Status, String> LABELS = Map.of(
```

Comment:
```suggestion
  private static final Map<SyncToRemoteStatus.Status, @Format({ConversionCategory.GENERAL}) String> LABELS = Map.of(
```

Reply:
(although this won't allow for `"untracked"` value in the map... see the comment below + use your judgement)

Reply:
I used `Match.Case` as suggested in other comment